src/libslic3r/GCode.cpp

```diff
@@ -5306,7 +5306,7 @@ std::string GCode::retract(bool toolchange, bool is_last_retraction, LiftType li
     return gcode;
 }
 
-std::string GCode::set_extruder(unsigned int extruder_id, double print_z)
+std::string GCode::set_extruder(unsigned int extruder_id, double print_z, bool is_init_extruder)
```

Comment:
There is variable called `m_toolchange_count` in GCode
We should be able to use that member variable to decide whether it's the first time to set extruder.
If that works then we don't need to add a new parameter here.
Do you mind to give that a try?

Reply:
🤦 
I'll make that change!